### DISCUSSION TITLE
Welcome to HowBadIsMyBatch Discussions!

### DISCUSSION THREAD

**Discussion by KnollFrank** [KnollFrank/HowBadIsMyBatch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhondtlo1** [comment]
How is the lethality and severity calculated?  I am trying to verify my numbers for Moderna Lot 011L20A.  HowBadIsMyBatch reports a lethality of 1.69% for that lot with 52 reported deaths.  If 52 makes up 1.69% of those reported, the batch would have consisted of 3077 doses.  There are 3,085 adverse reaction reports for the lot which comes out to more than the number of doses at a 1.69% lethality rate?  Should this be .169% or .0169%?  From what I can find on Moderna, their lots consist of ~500,000 doses.  With 52 deaths, wouldn't that make it approx. .000104%?  I was going to post your site in a report I am writing, but want to make sure I understand and can explain the numbers first.  Can you please elaborate on where I am going wrong?

**Reply by KnollFrank** [reply]
Hi,

the lethality is calculated [here](https://github.com/KnollFrank/HowBadIsMyBatch/blob/8e45d660c72e2184fb230528a7ccdb24422ba87d/src/SummationTableFactory.py#L17):

        Lethality = Deaths / 'Adverse Reaction Reports' * 100

which is then rounded to two decimal digits [here](https://github.com/KnollFrank/HowBadIsMyBatch/blob/8e45d660c72e2184fb230528a7ccdb24422ba87d/docs/batchCodeTable.js#L67). In your example it calculates

        Lethality = 52 / 3085 * 100 = 1.685575365 %

which is rounded to 1.69 % when displayed. So the problem you mentioned is due to rounding percentages to two decimal digits when displayed in the table on the web page.

Cheers, Frank

**Reply by dhondtlo1** [reply]
Thank you for the quick response.  I was thinking the percentages were based on the total number of doses given out of a batch.  Are those numbers available anywhere to include in the calculation?

**Reply by KnollFrank** [reply]
Unfortunately the number of doses isn't officially published as far as I know. Please see [here](https://www.howbadismybatch.com/lethal.html) for an explanation how lethality can be measured without knowing batch sizes.

**Reply by dhondtlo1** [reply]
Got it, thank you again. Had me worried at initial look.  When combining my two doses, it equates to a 5% lethality.  My thinking was that at 5%, 190 people from my unit that also were mandated to get the shots prior to deployment would have died, but, to my knowledge, nobody died or had a severe reaction at the unit.  It is still alarming numbers and glad that I didn't vaccinate our kids.  Thanks again for the explanation and for your work on this.

**Reply by KnollFrank** [reply]
Thank you and all the best to you and your children.

**Reply by JustSis987** [reply]
I have a similar question, and I don't mean to rehash. I do my own (little) podcast on these types of things - delving into stats sometimes when they're utterly conclusive. So I'm trying to understand. 
I did find your mathematical proof for lethality on https://www.howbadismybatch.com/lethal.html but you've lost me. You seem to be saying that the lethality percentage is "independent" of batch size. 
"The key point here is that this ratio is independent of batch size or of number of people vaccinated."
Well, it IS, but batch size must be known in order to figure out lethality. It, or lot size, depending on what the basis for slicing up the adverse events is, must literally be the denominator. 
Further, in this discussion, you say that lethality = deaths / adverse reaction reports, then turning that into a percent. 
That is not lethality. That is what percentage of all adverse events reported were deaths. Still a decent number for comparison across lots, but it would be many, many times what lethality in the standard sense would be.  First, you're leaving from the denominator any doses given where there were no adverse events. I know in non-deadened (non-overheated) lots, that would be rare, but it is still there. Second, you're leaving out the known aspect of underreporting of adverse events. I think you even cover that on your site.

I do think your great pull-together of numbers has values, but you're at the very least not defining things the way the typical person would take them or the way they would be considered in a standard way. 

Another example... you say that % of serious effects is independent of lot size. It is. But it doesn't make sense about how (at least for Pfizer's. I haven't found the moderna lookup yet) you say that a doses shipped that is twice as many as another for the same lot shows a "per 100,000" rate of exactly twice as much. If all we have is reporting down to lot number, than it could not be that we have the granular detail to say one subset of 2340 has twice the adverse effect rate of another subset of 1170. The per 100,000 should show the same rate. 

What am I missing? 

Also, for the moderna example the original poster brought up. if I'm looking in the right place (https://knollfrank.github.io/HowBadIsMyBatch/HowBadIsMyBatch.html ), I'm unsure where you get the 52 deaths. I looked for various stats on the right and see more like 38 or 39. 
To be clear, I'm utterly certain of the blood clots, the growths inside of the arteries, and the nearly immediate strokes. Because it's happening and because they admitted all of the severe side effects during a 10/22/20 national zoom call. Just wanting to make sure we don't have data or data interpretations out there that the "we are science" types can knock down easily, even as they ignore their own actual data.

**Reply by KnollFrank** [reply]
Sorry, I can't answer your question. Could you please email the author craig@howbad.info of the mentioned lethality document?

**Comment by oenothera12-gif** [comment]
I got batch Moderna 017c21a. I felt acid running from my left arm to my chest over and over and over. My heart was racing. I couldn’t breathe. I passed out right after the shot. My doctor diagnosed me with new onset asthma and new onset heart disease. I got shingles and blue hands and blue feet a little later. Thought I was dying. It felt like I got poisoned by a snake. Worst mistake of my life. My greatest regret! If my shot was mild compared to others I can’t even imagine worse. But ya!! I feel like I got some kind of venom batch.

**Reply by oenothera12-gif** [reply]
My question is did some people seem to get misfolded proteins (prions), some get acid or venom, and some a cancer compound and some parasites? I am pretty sure I got some kind of acid or venom.
